### PR TITLE
feat: US133511 - reduced motion for d2l-card

### PR DESCRIPTION
**Pull request by annabelsegalld2l**
No reduced motion set:
no-emulate

Before with reduced motion - looks very jittery 
before_reduce

After- don't move the card at all . Users can tell it is selected by the underlined link and shadow. 
reduced-mcw

### CODE REVIEW

File: components/card/card.js [hunk 1]
```diff
@@ -229,6 +229,13 @@ class Card extends RtlMixin(LitElement) {
 				:host {
 					transition: none;
 				}
+
+				:host(:hover),
+				:host([subtle]:hover),
+				:host([_active]:hover),
+				:host([subtle][_active]:hover) {
+					transform: none;
```
Comment (annabelsegalld2l) [inline components/card/card.js:237]: See transforms at lines 190 and 206.

**Review by GeorgePlukov — APPROVED** [review]

**Comment by GeorgePlukov** [comment]
LGTM, pulled and tested briefly as well

**Review by svanherk — APPROVED** [review]

**Comment by annabelsegalld2l** [comment]
Blocked - Waiting for 20.22.1 release branching to merge.

**Comment by annabelsegalld2l** [comment]
Reopening as we are now in the 20.22.1 release 🎉

**Review by dlockhart — APPROVED** [review]

**Pull request merged**